src/main/g8/play/src/main/resources/akka.conf

```diff
@@ -6,7 +6,9 @@ akka {
   logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
   # This is intentionally set to DEBUG - tune logging by adding filters to logback.xml file
   loglevel = "DEBUG"
-  stdout-loglevel = "OFF"
+  # Akka logs ONLY to stdout when the actor system is starting up and shutting down
+  # Setting this to OFF will silence those logs, but we will loose the visibility.
+  stdout-loglevel = "DEBUG"
```

Comment:
Why not just omit this entirely and leave it at the default (WARNING) - a happy medium somewhere between OFF and DEBUG?

Reply:
We set `loglevel = "DEBUG"` (line 8), so to be consistent I chose the same level.